tenant_settings.md

```diff
@@ -9,5 +9,5 @@ To add a settings for a specific tenant, use the following rake task: `rake db:t
 | bigbluebutton_secret | The secret for the BBB server to be used |
 | handler_params | The launch parameters to use for creating the room ID. To add more than one parameter, use the following syntax: "param1\\,param2". For example: "context_id\\,resource_link_id" |
 | enable_shared_rooms | Whether or not to enable the shared rooms feature. The default is false. |
-
+| hide_build_tag | Whether or not to hide the build tag feature. The default is true. |
```

Comment:
can you change the description to be: `Whether or not to hide the build tag. The default is false. `
(by default, if this setting is not set to true, then we show the build tag)